scripts/update_timestamp_explicit_client.sql

```diff
@@ -0,0 +1,13 @@
+-- UPDATE SCRIOT FOR CLIENT SCHEMA
+-- This script will set any null UpdateTimes to now
```

Comment:
For just the client schema? or for all schema (i.e. `server.sql`, `cloud.sql`, etc)?